repository_service_tuf/cli/admin/metadata.py

```diff
@@ -587,3 +605,117 @@ def update(
+    if response_data is None:
+        raise click.ClickException(response.text)
+
+    signing_roles: Dict[str, Any] = response_data.get("metadata", {})
+    if len(signing_roles) == 0:
+        raise click.ClickException("No metadata available for signing")
+
+    return signing_roles
+
+
+def _get_signing_key(role_info: MetadataInfo) -> RSTUFKey:
+    pending_keys: List = []
+    for key_id in role_info._new_md.signed.roles[Root.type].keyids:
+        if key_id not in role_info._new_md.signatures:
+            pending_keys.append(role_info._new_md.signed.keys[key_id])
+
+    sign_key_name = prompt.Prompt.ask(
+        "\nChoose a private key to load",
+        choices=[
+            signing_key.unrecognized_fields["name"]
+            for signing_key in pending_keys
+        ],
+    )
+
+    while True:
+        rstuf_key = _get_key(sign_key_name)
+        if rstuf_key.error:
+            console.print(rstuf_key.error)
+            retry = prompt.Confirm.ask(
+                f"\nRetry to load the key {sign_key_name}?"
+            )
+            if not retry:
+                raise click.ClickException("Aborted.")
+        else:
+            break
+
+    rstuf_key_id = rstuf_key.key["keyid"]
+
+    current_role_key = role_info._new_md.signed.keys[rstuf_key_id]
+    current_role_key_name = current_role_key.unrecognized_fields["name"]
+    if current_role_key_name != sign_key_name:
+        raise click.ClickException(f"Loaded key is not '{sign_key_name}'")
+
+    return rstuf_key
+
+
+def _sign_metadata(role_info: MetadataInfo, rstuf_key: RSTUFKey) -> Signature:
+    signer = role_info.get_signer(rstuf_key)
+    try:
+        signature = role_info._new_md.sign(signer)
+    except UnsignedMetadataError as err:
+        raise click.ClickException("Problem signing the metadata") from err
+
+    return signature
+
+
+@metadata.command()
+@click.option(
+    "--api-url",
+    help="URL or local path to the current root.json file.",
+    required=False,
+)
+@click.pass_context
+def sign(context, api_url: Optional[str]) -> None:
+    """
+    Start metadata signature.
+    """
+    console.print(markdown.Markdown(METADATA_SIGNING), width=100)
+
+    settings = context.obj["settings"]
+
+    signing_roles = _get_pending_signatures(settings, api_url)
+    rolename = prompt.Prompt.ask(
+        "\nChoose a metadata to sign", choices=[role for role in signing_roles]
+    )
+    role_info = MetadataInfo(Metadata.from_dict(signing_roles[rolename]))
+    console.print(
+        f"Signing [cyan]{rolename}[/] version "
+        f"{role_info._new_md.signed.version}"
+    )
+
+    rstuf_key = _get_signing_key(role_info)
+    signature = _sign_metadata(role_info, rstuf_key)
+
+    payload = {"role": rolename, "signature": signature.to_dict()}
+    console.print("\nSending signature")
+    task_id = send_payload(
+        settings,
+        URL.metadata_sign.value,
+        payload,
+        "Metadata sign accepted.",
+        "Metadata signature",
```

Comment:
The name of the command is not `signature` as we don't use it as a command, right?
```suggestion
        "metadata sign",
```